src/routes/indexRouter.go

```diff
@@ -0,0 +1,11 @@
+package routes
+

```

Comment:
file name can be index, instead of indexRouter, since we are already in router folder?

Reply:
ok